docker/drivers-build/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM busybox:1.36
```

Comment:
To reduce the size of the image. When i use `bitnami/spark` base the final image becomes more than a GB but if i use a minimal image as base the size becomes ~15mb.

I was thinking that since the only purpose of this image is to host all the drivers why not use a minimal image. It frees the need to maintain a parallel `spark` image.

If we use `bitnami/spark` as base we wont need to copy over the drivers but then we will need to maintain a parallel `spark`image.